### PR TITLE
Clarify optimization instructions

### LINKED ISSUE #4470: Unclear language regarding reuse of Materials for optimization.
**Your Godot version:**

**Issue description:**
https://docs.godotengine.org/en/latest/tutorials/performance/gpu_optimization.html#reuse-shaders-and-materials says

> If materials can't be reused, at least try to re-use shaders (or StandardMaterial3Ds with different parameters but the same configuration).

I'm unclear what the difference between "parameters" and "configuration" are. Can someone provide an example of materials with "different parameters but the same configuration" vs parameters with different configuration?

**URL to the documentation page:**
https://docs.godotengine.org/en/latest/tutorials/performance/gpu_optimization.html#reuse-shaders-and-materials

### PR DESCRIPTION
**Pull request by skyace65**
Clarifies what a parameter and configuration are in a standard material. Closes #4470

### CODE REVIEW

File: tutorials/performance/gpu_optimization.rst [hunk 1]
```diff
@@ -87,8 +87,9 @@ possible. Godot's priorities are:
    of objects (in the hundreds or thousands), try reusing the materials.
    In the worst case, use atlases to decrease the amount of texture changes.
 -  **Reusing Shaders:** If materials can't be reused, at least try to
-   re-use shaders (or StandardMaterial3Ds with different parameters but the same
-   configuration).
+   re-use shaders. Or have StandardMaterial3Ds with different parameters (things
+   that can change without enabling or disabling features) but the same
+   configuration(things enabled or disabled with a check box).
```
Comment (clayjohn) [inline tutorials/performance/gpu_optimization.rst:92]:
```suggestion
   reuse shaders. Note: shaders are automatically reused between
   StandardMaterial3Ds that share the same configuration (features
   that are enabled or disabled with a check box) even if they have different
   parameters.
```

Comment (clayjohn) [reply tutorials/performance/gpu_optimization.rst:92]: Just a little bit of clarification :)

**Comment by skyace65** [comment]
Fixed.

**Review by clayjohn — APPROVED** [review]

**Comment by clayjohn** [comment]
Thank you!

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.2 and 3.3.

**Pull request merged**